### ISSUE TITLE
Subset DB to function.ids

### ISSUE THREAD

**Issue by jakob-r** [mboecker/lookup-server#39]
Like here:
https://github.com/ja-thomas/OMLbots/blob/e44101679df36c46cefae81f33a40789b826805b/snapshot_database/database_extraction.R#L13

**Comment by mboecker** [comment]
Like
```sql
DELETE FROM evaluation WHERE function_id NOT IN (4,45,54,59,63);
```

**Comment by mboecker** [comment]
Closed by 4fc9303adba1fd2106700e741b6a564a7e476e75
